apps/dashboard/src/app/account/account-status/account-status.component.ts

```diff
@@ -26,29 +25,33 @@ export class AccountStatusComponent {
   @Input() usage?: AccountUsage;
 
   accountType = this.sdk.currentAccount.pipe(map((account) => account.type));
-  trialAccount = this.accountType.pipe(map((type) => type === 'stash-trial'));
+  isTrial = this.accountType.pipe(map((type) => type === 'stash-trial'));
   currency = this.billingService.getAccountUsage().pipe(map((usage) => usage.currency));
   price = combineLatest([this.billingService.getPrices(), this.accountType]).pipe(
     map(([prices, accountType]) => prices?.[accountType]?.recurring.month.price),
   );
   upgradeUrl = this.sdk.currentAccount.pipe(
     map((account) => this.navigation.getAccountManageUrl(account.slug) + '/billing/subscriptions'),
   );
-  blocked = this.sdk.currentAccount.pipe(map((account) => account.blocking_state === AccountBlockingState.QUOTA));
   isBillingEnabled = this.tracking.isFeatureEnabled('billing').pipe(shareReplay(1));
   daysLeft = this.sdk.currentAccount.pipe(
     filter((account) => !!account.trial_expiration_date),
     map((account) => {
       const expiration = new Date(`${account.trial_expiration_date}+00:00`);
       const now = new Date();
-      return differenceInDays(expiration, now);
+      return differenceInDays(expiration, now) + 1;
     }),
   );
-  canExtendTrial = combineLatest([this.daysLeft, this.trialAccount]).pipe(
+  canExtendTrial = combineLatest([this.daysLeft, this.isTrial]).pipe(
     map(([daysLeft, isTrial]) => isTrial && daysLeft <= 5),
   );
-  trialExpired = combineLatest([this.daysLeft, this.trialAccount]).pipe(
-    map(([daysLeft, isTrial]) => isTrial && daysLeft < 0),
+  trialExpired = combineLatest([this.sdk.currentAccount, this.isTrial]).pipe(
+    map(([account, isTrial]) => {
+      if (!isTrial || !account.trial_expiration_date) return false;
+      const expiration = new Date(`${account.trial_expiration_date}+00:00`);
```

Comment:
I know why you did this trick of adding  `+00:00` at the end of the date in several places, but you should not do that. it's backend's job to send date properly formatted (= with timezone). I already told @albertnadal and @sunbit several times when I was working on the manager, so any time you notice timestamp not formatted as expected, ping them so they fix it on the backend. But **do not** add this kind of kix on the frontend part otherwise we will have bad bugs as soon as the backend will be fixed…

Reply:
ok, I'll let them know